.gitignore

```diff
@@ -1 +1 @@
-/node_modules
+node_modules/
```

Comment:
We should keep the leading slash because it's only the top-level node_modules directory that should be ignored. (Not that it would happen, but in general, we'd only want to ignore these items at top-level.)